Client/Frontend/Shields/ShareShieldsController.swift

```diff
@@ -202,13 +202,27 @@ private class ShareTypeView: UIControl, Themeable {
         $0.font = .systemFont(ofSize: 19)
         $0.numberOfLines = 0
         $0.setContentHuggingPriority(.required, for: .horizontal)
+        $0.isAccessibilityElement = true
     }
 
     private let titleImageView = UIImageView().then {
         $0.contentMode = .scaleAspectFit
         $0.tintColor = .darkGray
         $0.setContentCompressionResistancePriority(.required, for: .horizontal)
     }
+    
+    override func accessibilityElementCount() -> Int {
+        return 1
+    }
+
+    override func accessibilityElement(at index: Int) -> Any? {
+        switch index {
+        case 0:
+            return titleLabel
+        default:
+            return nil
+        }
+    }
```

Comment:
How come this is needed? I've never needed this personally to setup a count + element

Reply:
The view has a container view and inside there are some elements. So what I wanted to achieve here is use title label text as accessibilityElement regardless of what has been added inside and will be added later.